docs/csharp/tour-of-csharp/tutorials/hello-world.yml

```
@@ -62,7 +62,7 @@ items:
     of the `aFriend` variable and its initial assignment.
 
     ```csharp
-    aFriend = "Maira";
+    string aFriend = "Maira";
```

And then, let's revert this change:

```suggestion
    aFriend = "Maira";
```